### PR TITLE
Feature/intro

### PR DESCRIPTION
**Pull request by pengin7384**
Complete
- Change background image
- Change background width

### CODE REVIEW

**Comment by pengin7384** [comment]
새로운 인트로 페이지도 작업하여 추가하였습니다~ PTAL @minsulee2

File: src/components/page/AppPage.js [hunk 1]
```diff
@@ -59,18 +61,31 @@ class AppPage extends React.Component {
             <Wrapper>
                 { this.state.clicked ? <Spinner/>: ""}
 
-                <TextWrapper>
-                    <GradationText style= {{marginTop:"56px"}} fontSize={2.6} fontWeight={'bold'}>
-                        PythonQuizFlex
+                <TextWrapper style={{marginTop: '5%'}}>
+                    <img style={{
```
Comment (minsulee2) [inline src/components/page/AppPage.js:65]: styled image would be better

File: src/components/page/AppPage.js [hunk 2]
```diff
@@ -59,18 +61,31 @@ class AppPage extends React.Component {
...
+                        width: '40px',
+                        height: '40px',
+                        marginLeft: 'auto',
+                        marginRight: 'auto',
+                        marginBottom: '28px',
+                    }}
+                         src={require('../../static/img/intro/python_logo.png')}
+                         alt="Python logo"/>
+
+                    <GradationText style={{
```
Comment (minsulee2) [inline src/components/page/AppPage.js:75]: Please make it styled component

File: src/components/page/AppPage.js (hunk 1 above)
Comment (pengin7384) [reply src/components/page/AppPage.js]: Thx Done!

File: src/components/page/AppPage.js (hunk 2 above)
Comment (pengin7384) [reply src/components/page/AppPage.js]: Done!

**Comment by pengin7384** [comment]
@minsulee2 PTAL

File: src/components/page/AppPage.js [hunk 3]
```diff
@@ -60,17 +85,18 @@ class AppPage extends React.Component {
                 { this.state.clicked ? <Spinner/>: ""}
 
                 <TextWrapper>
-                    <GradationText style= {{marginTop:"56px"}} fontSize={2.6} fontWeight={'bold'}>
-                        PythonQuizFlex
-                    </GradationText>
+                    <LogoImg/>
+
+                    <GradationTitle>
```
Comment (minsulee2) [inline src/components/page/AppPage.js:90]: Can it be styled component?

File: src/components/page/ResultPage.js [hunk 4]
```diff
@@ -141,12 +141,12 @@ class Result extends React.Component {
                     <meta property="og:description" content={`${description}`} />
                     <meta property="og:image" content={require(`../../static/img/result/level_${score}.png`)} />
                 </Helmet>
-                
+
                 <LevelText>
                     Your level is
```
Comment (minsulee2) [inline src/components/page/ResultPage.js:146]: {} please

File: src/components/page/ResultPage.js [hunk 5]
```diff
@@ -164,17 +164,17 @@ class Result extends React.Component {
                     {result.Description}
                 </Description>
 
-                <GradationText fontSize={1}>
+                <GradationText fontSize={'1em'}>
                     Flex your level
```
Comment (minsulee2) [inline src/components/page/ResultPage.js:168]: {} please

File: src/components/page/AppPage.js (hunk 3 above)
Comment (pengin7384) [reply src/components/page/AppPage.js:90]: I deleted the GradationTitle and changed the direction to reuse the GradationText! Done

File: src/components/page/ResultPage.js (hunk 5 above)
Comment (pengin7384) [reply src/components/page/ResultPage.js:168]: Done

File: src/components/page/ResultPage.js (hunk 4 above)
Comment (pengin7384) [reply src/components/page/ResultPage.js]: Done!

**Pull request merged**